agents/rust/mediator/mediation/src/server.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2023 Naian G.
+// SPDX-License-Identifier: Apache-2.0
+use log::info;
+
+use crate::router::create_router;
+
+pub async fn run_server() {
+    // app definition, and routings
+    let app = create_router().await;
+    info!("Starting server");
+    // add server task to main loop
+    axum::Server::bind(&"127.0.0.1:7999".parse().unwrap())
```

Comment:
Let's the IP and the port configurable. You can keep the current values as defaults